### DISCUSSION TITLE
World Cup is here! let's gear up

### DISCUSSION THREAD

**Discussion by Galphath** [sunflower-land/sunflower-land#1680] [Art]
Hello! with the world cup just about to start it would be fantastic to get ready for it and show our colors! 

Farm ID: 170

 **--- Humans**

![Human_sample1](https://user-images.githubusercontent.com/20891096/202360781-774983d4-6d0a-4330-ba63-c6c11b08907c.png)
![Human_sample2](https://user-images.githubusercontent.com/20891096/202360783-e1bfd2b7-864f-4f5d-8aa5-517a78a5a1e9.png)

![Human_pants](https://user-images.githubusercontent.com/20891096/202360815-b0be70a6-efcf-4ca2-afaf-90cde75b54f9.png)
![Human_shirt](https://user-images.githubusercontent.com/20891096/202360818-1bcf5d06-2028-4757-96bc-9b9fec301901.png)
![Human_banner](https://user-images.githubusercontent.com/20891096/202360839-ca3ff935-1ad7-427d-9ceb-a57d5f6417f1.png)
![Human_socks](https://user-images.githubusercontent.com/20891096/202360843-98f9e333-fc2a-4cc6-bb96-5d301e2e28fe.png)

**--- Goblins**
![Goblin_sample2](https://user-images.githubusercontent.com/20891096/202360889-49ad92c4-9340-4416-9cef-13e7a0d834bd.png)
![Goblin_sample1](https://user-images.githubusercontent.com/20891096/202360891-6bff9373-2141-47b0-8ad9-728c8c2f256a.png)

![Goblin_pants](https://user-images.githubusercontent.com/20891096/202360925-bf4f0988-1064-469d-8bf3-e09df6cc8b3c.png)
![Goblin_shirt](https://user-images.githubusercontent.com/20891096/202360927-dc78e372-e7e9-49e7-8a7e-7c4b920fdf58.png)
![Goblin_Banner](https://user-images.githubusercontent.com/20891096/202360955-eb5eb99a-a6a0-4888-b4b7-7f647a7bb3fe.png)
![Goblin_socks](https://user-images.githubusercontent.com/20891096/202360957-0d914f7d-520e-40c0-9f10-c6c3dc51b986.png)

** --- Common**
![Trophy_arm](https://user-images.githubusercontent.com/20891096/202361039-6aa73f52-e890-429e-89eb-8051ab7daa68.png)
![Trophy_full](https://user-images.githubusercontent.com/20891096/202361042-b576d7b4-2832-47f8-ad9c-e516cc7ae6c4.png)

![Football](https://user-images.githubusercontent.com/20891096/202361044-c2ea7791-bbad-474a-8c23-a2cabf295cd4.png)
![Football_arm](https://user-images.githubusercontent.com/20891096/202361045-55c98956-f5bd-4233-91ea-e4b8fda0422a.png)
![Football_shoes](https://user-images.githubusercontent.com/20891096/202361047-60e5b2bc-5453-4a8d-bbd3-967bcdadf917.png)
![Football_speech_bubble](https://user-images.githubusercontent.com/20891096/202361048-1a325ba1-3112-4b17-a7d8-ad7bfa9f17b9.png)
![Stadium_bg](https://user-images.githubusercontent.com/20891096/202361055-f7683120-da44-4787-a658-069fe0145e1e.png)

I'm aware that with land expansion upon us a big event would be too much but maybe something on discord like giving away shirts and stuff for team human and team goblin while any of the cup's matches is on could be nice.

![image](https://user-images.githubusercontent.com/20891096/202361971-5d97d8e9-747a-49e4-a794-0f28a57a5498.png)
Thanks for watching!

Coming soon : those hairstyles, on different colours on a different discussion

**Comment by Ang0212** [comment]
Muy buena idea abue!
